### PR TITLE
Add support for importing troi generated playlist excerpts

### PR DESCRIPTION
**Pull request by mayhem**
This PR is based off @amCap1712's year-in-music branch and should first be merged into that branch!

This PR adds support for taking playlist excerpt dumps created by troi and import them into the LB database. Each dump should be formatted as follows:

<user_name>\n
<playlist_mbid>\n
<playlist_JSPF>\n

To import the file use:

manage.py dump import_yim_playlists

### CODE REVIEW

File: listenbrainz/db/year_in_music.py [hunk 1]
```diff
@@ -160,3 +161,44 @@ def handle_yearly_listen_counts(data):
...
+        while True:
+            user_name = f.readline()
+            if user_name == "":
+                break
+
+            user_name = user_name.strip()
+            playlist_mbid = f.readline().strip()
+            jspf = f.readline().strip()
+
+            data.append((user_name, ujson.dumps({troi_patch_slug: {"mbid": playlist_mbid, "jspf": jspf}})))
```
Comment (amCap1712) [inline listenbrainz/db/year_in_music.py:191]:
```suggestion
            data.append((user_name, ujson.dumps({troi_patch_slug: {"mbid": playlist_mbid, "jspf": ujson.loads(jspf)}})))
```

File: listenbrainz/db/dump_manager.py [hunk 2]
```diff
@@ -320,6 +322,23 @@ def create_test_parquet_files():
 
         sys.exit(-2)
 
+
+@cli.command(name="import_yim_playlists")
+@click.argument('patch-slug', type=str)
+@click.argument('dump-file', type=str)
+def create_full(patch_slug, dump_file):
```
Comment (amCap1712) [inline listenbrainz/db/dump_manager.py:329]: Rename the method.

Comment (amCap1712) [reply listenbrainz/db/dump_manager.py:329]: Probably also add a comment that first copy the file to container.

File: listenbrainz/db/year_in_music.py (hunk 1 above)
Comment (mayhem) [reply listenbrainz/db/year_in_music.py]: Oh doh.

File: listenbrainz/db/dump_manager.py (hunk 2 above)
Comment (mayhem) [reply listenbrainz/db/dump_manager.py]: Makes sense. Can you please take care of this while you're at it?

**Comment by mayhem** [comment]
Looks good now -- please remove the prints and then merge.

File: listenbrainz/db/dump_manager.py (hunk 2 above)
Comment (amCap1712) [reply listenbrainz/db/dump_manager.py]: Already done :)

**Pull request merged**